spec/integration/_dev/benchmark/spec.yml

```diff
@@ -2,9 +2,9 @@ spec:
   additionalContents: false
   developmentFolder: true
   contents:
-    - description: Folder containing end-to-end benchmarks scenarios.
+    - description: Folder containing system benchmarks scenarios.
       type: folder
-      name: e2e
+      name: system
```

Comment:
This change will be breaking, did we mark this feature as beta/experimental? Is it currently used?
To avoid breaking changes we could duplicate the definition and keep both working...

Reply:
No currently used, it should be alright to break it as it is first used in https://github.com/elastic/elastic-package/pull/1232